widget.json

```diff
@@ -12,11 +12,15 @@
     "dependencies": [
       "vue.js",
       "fliplet-core",
-      "fliplet-studio-ui"
+      "fliplet-studio-ui",
+      "highcharts",
+      "font-awesome"
     ],
     "assets": [
       "dist/css/index.css",
-      "dist/app.js"
+      "dist/app.js",
+      "vendors/poper.min.js",
+      "vendors/bootstrap.js"
```

Comment:
I think we have the `bootstrap-js` dependency that you can add, @tonytlwu can confirm.

Reply:
Correct

Reply:
Yes, you are right about dependency it worked great.
Now we are using it and I have removed vendors folder from the project as we didn't need it anymore.